package.json

```diff
@@ -55,7 +55,7 @@
     "data-spreadsheet"
   ],
   "dependencies": {
-    "handsontable": "0.38.1",
```

Comment:
Could you change the `handsontable` dependency to `develop`, so it won't accidentally mess up the tests?